README.md

```diff
@@ -1,60 +1,63 @@
-# Introduction #
+<p align="center">
+  <img width="460" src="doc/logo/psyclone_v1.0.png">
+</p>
 
-Welcome to PSyclone. PSyclone is a code generation and transformation
-system that generates appropriate code for the PSyKAl code structure
-developed in the GungHo project. It is also capable of working with
-existing Fortran code.
+![Build Status](https://github.com/stfc/PSyclone/workflows/PSyclone%20tests%20and%20examples/badge.svg)
+[![codecov](https://codecov.io/gh/stfc/PSyclone/branch/master/graph/badge.svg)](https://codecov.io/gh/stfc/PSyclone)
 
-Please see [psyclone.pdf](psyclone.pdf) in this directory (or on
-[ReadTheDocs](http://psyclone.readthedocs.io)) for
-more information. If you would prefer to build the documentation,
-please see the [README](doc/README.md) file in the "doc" directory.
+PSyclone is a source-to-source Fortran compiler designed to programmatically
+optimise and parallelise HPC applications using user-provided transformation
+scripts. These scripts offer a separation of concerns between the science code
+and the performance portability implementation choices, allowing to experiment
+and develop with each side independenly. Additonally, PSyclone supports the
+development of kernel-based Fortran-embedded DSLs following the PSyKAl model
+developed in the [GungHo project](https://www.metoffice.gov.uk/research/foundation/dynamics/next-generation).
 
-# Try it on Binder #
+PSyclone is currently used to support the
+[LFRic PSyKAl mixed finete elements DSL](https://www.metoffice.gov.uk/research/modelling-systems/lfric/)
+for the UK MetOffice's next generation modelling system and the
+[GOcean finite difference prototype DSL](https://gtr.ukri.org/projects?ref=NE%2FL01209X%2F1).
+It is also used to insert GPU offloading directives to existing
+directly-addressed MPI applications such as the [NEMO ocean model](https://www.nemo-ocean.eu/).
```

Comment:
Again, a link to the UG too?

Reply:
Done, although the Nemo docs are due for a refactor, but having the link it will remind us to update it in the future.